velox/dwio/parquet/tests/reader/E2EFilterTest.cpp

```diff
@@ -59,14 +59,19 @@ class E2EFilterTest : public E2EFilterTestBase {
     auto sink = std::make_unique<MemorySink>(*leafPool_, 200 * 1024 * 1024);
     sinkPtr_ = sink.get();
     options_.memoryPool = rootPool_.get();
-
     options_.bufferGrowRatio = 2;
+    auto flushPolicyFactory = [this]() {
```

Comment:
Can we adjust the flush behavior in test to be the same as DWRF?
https://github.com/facebookincubator/velox/blob/7422f5fdb0144f06dc9ba4c777075a5188f842ed/velox/dwio/dwrf/test/E2EFilterTest.cpp#L70-L75

Reply:
I mean can we make it obey the behavior for `forRowGroupSkip` and `flushEveryNBatches_`?

Reply:
oh, sorry for the misunderstanding. I think I have addressed the comment now.